test/integration/main_test.go

```diff
@@ -358,6 +358,149 @@ deletion_timeline:
 	}
 }
 
+// TestMappingReusableWorkflowCall tests the mapping file-copy reusable workflow has successfully
+// uploaded the file to gcs and triggered pmap event handler, and write the corresponding entry
+// into bigquery.
+func TestMappingReusableWorkflowCall(t *testing.T) {
+	t.Parallel()
+	cases := []struct {
+		name                string
+		bigqueryTable       string
+		traceID             string
+		wantResourceMapping *v1alpha1.ResourceMapping
+	}{
+		{
+			name:          "mapping-copy-success-event",
+			bigqueryTable: cfg.MappingTableID,
+			traceID:       "fake-pmap-dev-manual-test-mapping",
```

Comment:
Can you just set the value of traceID to the workflow run id and then we can make the query string consistent?

Reply:
Discussed offline, the traceID is from the yaml file, which makes it a lot of extra work to enable this.